docs/knowledge/koor/osd-scrubbing-schedule.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "OSD Scrubbing Schedule"
+---
+
+!!! hint
+    This is a paid Koor Storage Distribution feature.
+
+This is an example of the OSD scrubbing config in your CephCluster object:
+
+```yaml
+spec:
+  # [...]
+  storage:
+    # [...]
+    scrubbing:
+      # Whether the scrubbing schedule specified should be applied to the cluster.
+      applySchedule: true
+      # Max scrubbing operations happening at the same time. Default: `3`.
+      maxScrubOps: 3
+      # Begin hour of scrubbing schedule. Default `0`. Setting both `BeginHour` and `EndHour` to `0`, will allow scrubbing the entire day.
+      beginHour: 8
+      # End hour of scrubbing schedule. Default `0`.
+      endHour: 17
+      # Begin week day of scrubbing schedule. `0` = Sunday, `1` = Monday, etc. Default: `0`.
+      beginWeekDay: 1
+      # End week day of scrubbing schedule. `0` = Sunday, `1` = Monday, etc. Default: `0`.
+      endWeekDay: 5
+      # Minimum interval to wait before scrubbing again. Default: `24h` (1 day).
+      minScrubInterval: 24h
+      # Maximum interval to wait before scrubbing is forced. Default: `168h` (7 days).
+      maxScrubInterval: 168h
+      # Interval at which to do deeb scrubbing instead of a "light" scrubbing. Default: `168h` (7 days).
+      deepScrubInterval: 168h
+      # Set the scrub sleep as a duration. Default: `0ms`, if you are impacted by scrubbing causing performance issues, it is recommended to set it to at least `100ms`.
+      scrubSleepSeconds: 100ms
+```
+
+## Specifications
+
+These options can be used to set a custom OSD scrubbing schedule easily instead of having to use the Ceph config override:
```

Comment:
this can be used right after the title to give some insights on what this is about

Reply:
I have added a different intro sentence to the top of the page, PTAL